frontend/src/components/pages/ImportStoryPage.tsx

```diff
@@ -0,0 +1,184 @@
+import Select from "react-select";
+import {
+  Box,
+  Button,
+  Flex,
+  FormControl,
+  FormLabel,
+  Heading,
+  Input,
+  Link,
+  Text,
+  Textarea,
+} from "@chakra-ui/react";
+import { Icon } from "@chakra-ui/icon";
+import { MdArrowBackIosNew, MdFolder } from "react-icons/md";
+import DropdownIndicator from "../utils/DropdownIndicator";
+import Header from "../navigation/Header";
+import { colourStyles } from "../../theme/components/Select";
+
+const ImportStoryPage = () => {
+  return (
+    <Flex direction="column" height="100vh" justifyContent="space-between">
+      <Header />
+      <Flex
+        direction="column"
+        flex="1"
+        padding="40px 120px 40px 120px"
+        overflow="auto"
+      >
+        <Link
+          color="blue.100"
+          fontWeight="bold"
+          href="#/"
+          textDecoration="none"
+          _hover={{ textDecoration: "none", color: "blue.100" }}
+        >
+          <Icon as={MdArrowBackIosNew} margin="-2px 10px 0 0" />
+          Back to admin page
+        </Link>
+        <Heading margin="35px 0 30px 0">Import a New Book</Heading>
+        <Flex>
+          <Flex direction="column" paddingRight="40px" width="50%">
+            <FormControl isRequired>
+              <FormLabel fontWeight="bold" htmlFor="title" marginBottom="15px">
+                Title
+              </FormLabel>
+              <Input id="title" size="md" type="text" />
+            </FormControl>
+            <FormControl isRequired marginTop="40px">
+              <FormLabel
+                fontWeight="bold"
+                htmlFor="description"
+                marginBottom="15px"
+              >
+                Description
+              </FormLabel>
+              <Textarea id="description" height="150px" type="text" />
+            </FormControl>
+            <FormControl marginTop="40px">
+              <FormLabel fontWeight="bold" htmlFor="level" marginBottom="15px">
+                Level
+              </FormLabel>
+              <Box width="300px">
+                <Select
+                  components={{ DropdownIndicator }}
+                  id="level"
+                  placeholder="Select level"
+                  styles={colourStyles}
+                />
+              </Box>
+            </FormControl>
+            <FormControl isRequired marginTop="40px">
+              <FormLabel
+                fontWeight="bold"
+                htmlFor="youtube-link"
+                marginBottom="15px"
+              >
+                Youtube Link
+              </FormLabel>
+              <Input id="youtube-link" size="md" type="text" />
+            </FormControl>
+            <FormControl marginTop="40px">
+              <FormLabel
+                fontWeight="bold"
+                htmlFor="languages"
+                marginBottom="15px"
+              >
+                Languages to Exclude
+              </FormLabel>
+              <Box width="300px">
+                <Select
```

Comment:
can you add `options` and `getOptionLabel` like in `TableFilter.tsx` or `UserProfileForm.tsx`? Youll need to 
```
import { languageOptions } from "../../constants/Languages";
import { convertLanguageTitleCase } from "../../utils/LanguageUtils";
```
and then on the select
```
options={languageOptions}
getOptionLabel={(option: any) => `
    ${convertLanguageTitleCase(option.value || "")}
`}
```